db/migrate/20181101185901_create_organization_webhooks.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOrganizationWebhooks < ActiveRecord::Migration[5.1]
+  def change
+    create_table :organization_webhooks do |t|
+      t.integer :github_webhook_id
+      t.datetime :last_webhook_recieved
+
```

Comment:
Should this table also keep a record of the `github_organization_id`? Or are we planning to build a model for that?